<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoBasicFeatures</class>
 <widget class="QWidget" name="DemoBasicFeatures">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test basic features</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="PwGisWidget" name="pwgiswidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btnOpenMap">
           <property name="text">
            <string>Open Map</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCloseMap">
           <property name="text">
            <string>Close Map</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddMarker">
           <property name="text">
            <string>add marker</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddPoint">
           <property name="text">
            <string>add point</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddLine">
           <property name="text">
            <string>add line</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddRectangle">
           <property name="text">
            <string>add rectangle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddCircle">
           <property name="text">
            <string>add circle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddPath">
           <property name="text">
            <string>add path</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddPolygon">
           <property name="text">
            <string>add polygon</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddClassicPolygon">
           <property name="text">
            <string>add classic polygon</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddSlice">
           <property name="text">
            <string>add slice</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddRing">
           <property name="text">
            <string>addRing</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddRingSegment">
           <property name="text">
            <string>addRingSegment</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeleteAllFeatures">
           <property name="text">
            <string>delete all features</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lblFeatureId">
         <property name="text">
          <string>feature ID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtFeatureId"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnGetGeometry">
         <property name="text">
          <string>get geometry</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDeleteFeature">
         <property name="text">
          <string>delete feature</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="txtLog">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PwGisWidget</class>
   <extends>QWidget</extends>
   <header location="global">PwGis/pwgiswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
